apps/api/src/app/workflows/dto/update-workflow-request.dto.ts

```diff
@@ -37,9 +37,6 @@ export class UpdateWorkflowRequestDto implements ICreateNotificationTemplateDto
   @IsMongoId()
   notificationGroupId: string;
 
-  @ApiPropertyOptional()
-  active?: boolean;
-
```

Comment:
This is an example of why this refactor was needed- a change/fix for the notification templates needed to be duplicated in the workflows folder.